cmd/utils/global_options.go

```diff
@@ -86,3 +86,23 @@ func ChannelAccountEncryptionPassphraseOption(configKey *string) *config.ConfigO
 		Required:  true,
 	}
 }
+
+func SentryDSNOption(configKey *string) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:      "tracker-dsn",
+		Usage:     "The Sentry DSN",
+		OptType:   types.String,
+		ConfigKey: configKey,
+		Required:  true,
+	}
+}
+
+func StellarEnvironmentOption(configKey *string) *config.ConfigOption {
```

Comment:
Sentry needs a variable called Environment when you call sentry.Init(...) and this basically sets the value for that

Reply:
It is used to set up the sentry environment which is a variable we need to initialize the sentry tracker

Reply:
I think we can use the `Network Passphrase` for that, what do you think?

Reply:
Seems like we use the sentry environment variable to initialize the tracker in the Vibrant backend so I did the same here

Reply:
@gouthamp-stellar this is something we're refactoring on the Vibrant backend. Unfortunately we have this duplicate ENV there.
Wdyt about using the Network Passphrase env that we already use for ingestion (like @CaioTeixeira95 suggested) and mapping it to `testnet/development` or `mainnet/production` in the `NewSentryTracker` function.
That way we don't have this duplicate config which at the end of the day means the same thing.

Reply:
If not maybe we could name the env TRACKER_ENVIRONMENT so it's not misused in other places.

Reply:
Down to do that in a future pr. I already pushed a pr to kube-vibrant with this env variable. Since it's only a couple lines, I can add this change to an upcoming pr